### PR TITLE
feat: add some RSS API methods

### PR DESCRIPTION
**Pull request by skrmc**
Added the following RSS API methods:

- [x]  Add folder
- [x]  Add feed
- [x]  Remove item
- [x]  Move item
- [ ]  Get all items
- [x]  Mark as read
- [x]  Refresh item
- [ ]  Set auto-downloading rule
- [x]  Rename auto-downloading rule
- [x]  Remove auto-downloading rule
- [ ]  Get all auto-downloading rules
- [ ]  Get all articles matching a rule

This pull request does not include any new struct, only relatively simple operations. 
All functions should have been tested, but if there are any issues please let me know.

Thanks a lot for your work!

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name          = "qbit-rs"
-version       = "0.4.1"
```
Comment (George-Miao) [inline Cargo.toml:3]: Please don't bump the version yet.

File: README.md [hunk 2]
```diff
@@ -126,16 +126,16 @@ Most of the API is covered, except `RSS` and `Search`. PR is welcomed if you nee
    1. [x] [Rename file](https://docs.rs/qbit-rs/latest/qbit_rs/struct.Qbit.html#method.rename_file)
    1. [x] [Rename folder](https://docs.rs/qbit-rs/latest/qbit_rs/struct.Qbit.html#method.rename_folder)
 1. [ ] RSS (experimental)
-   1. [ ] Add folder
-   1. [ ] Add feed
-   1. [ ] Remove item
-   1. [ ] Move item
+   1. [x] Add folder
```
Comment (George-Miao) [inline README.md]: Add urls like other entries.

**Review by George-Miao — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (skrmc) [reply Cargo.toml:3]: Sure, I just thought 0.4.2 is the current version. Would you like to keep it 0.4.1 here?

**Review by George-Miao — APPROVED** [review]
LGTM, thanks for the effort!

**Pull request merged**